src/components/themed/WalletListRow.js

```diff
@@ -181,9 +181,10 @@ class WalletListRowComponent extends React.PureComponent<Props & ThemeProps, Sta
               <View style={styles.detailsContainer}>
                 <View style={styles.detailsRow}>
                   <EdgeText style={styles.detailsCurrency}>{currencyCode}</EdgeText>
-                  <EdgeText style={[styles.exchangeRate, { color: differencePercentageStyle }]}>
-                    {exchangeRateFiatSymbol + exchangeRate + '     ' + differencePercentage}
-                  </EdgeText>
+                  <View style={styles.exchangeRateContainer}>
+                    <EdgeText style={{ color: differencePercentageStyle, marginRight: theme.rem(0.5) }}>{exchangeRateFiatSymbol + exchangeRate}</EdgeText>
```

Comment:
No need to make a separate component. The previous code works and all you need to do is remove the excessive spacing you added.

Just change line 185 to

`{exchangeRateFiatSymbol + exchangeRate + ' ' + differencePercentage}`